src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -262,8 +262,25 @@ const orderedSlides = computed(() => {
 	return sortedSlides;
 });
 
+const optimizeContentfulUrl = baseUrl => {
+	if (!baseUrl) return '';
+
+	// journey card dimensions: responsive sizing
+	const width = isMobile.value ? 320 : 336;
+	const height = isMobile.value ? 340 : 382;
```

Comment:
Are we sure about these sizes? The journey card seems to render at 336x340 on desktop

<img width="393" height="479" alt="Image" src="https://github.com/user-attachments/assets/152b6a0c-03b3-4afa-a881-42e3c25bc2eb" />